### PR TITLE
Add release notes february 2023

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/release-notes/february-2023.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+toc_max_heading_level: 3
+---
+
+# February 2023
+
+## Important update
+
+- Semgrep CLI is now officially renamed to **Semgrep OSS Engine**. Note that Semgrep CLI is an interface, not a product. As an interface, you can use the CLI as an interface for several products. For example the OSS, Semgrep Code, Semgrep Supply Chain.
+- Team tier rules are now renamed to Pro Rules. The **Pro rules** are created by r2c and targeted for security and software engineers who need accurate findings. These rules were previously called Team tier rules. As of this update, these rules are officially called the **[Pro rules](/semgrep-code/pro-rules/)** and are available with the [Team or higher tier](https://semgrep.dev/pricing).
+- DeepSemgrep is renamed to **Semgrep Pro Engine**. Semgrep Pro Engine is fully available for [Team or higher tier](https://semgrep.dev/pricing) users. See [Semgrep Pro](/deepsemgrep/deepsemgrep-introduction/) documentation. Some specific flags have been removed:
```
Comment (bmahe) [inline docs/release-notes/february-2023.md:16]: nitpick: **Semgrep Pro Engine** contains more than just DeepSemgrep.
So maybe we should rephrase it to "DeepSemgrep has been merged into **Semgrep Pro Engine**"
But that's really a nitpick

**Review by bmahe — APPROVED** [review]

File: docs/release-notes/february-2023.md (hunk 1 above)
Comment (adamkvitek) [reply docs/release-notes/february-2023.md:16]: @bmahe more such nitpicks, please! Thank you. :)

Comment (adamkvitek) [reply docs/release-notes/february-2023.md:16]:
```suggestion
- DeepSemgrep has been bundled with other functionalities under the name of **Semgrep Pro Engine**. Semgrep Pro Engine is fully available for [Team or higher tier](https://semgrep.dev/pricing) users. See [Semgrep Pro](/deepsemgrep/deepsemgrep-introduction/) documentation. Some specific flags have been removed:
```

File: docs/semgrep-app/findings.md [hunk 2]
```diff
@@ -136,11 +136,15 @@ Display and filter for findings reported only in specific time. To display findi
...
 ### Grouping by rule
 
-Semgrep Cloud Platform by default groups findings by a specific rule on the Findings page. This view can be disabled.
+Semgrep Cloud Platform by default groups findings by a specific rule on the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page.
 
 ![Screenshot of the Findings page with findings grouped by rule](/img/app-findings.png)<br />
 *Figure 3.* Screenshot of the Findings page with findings grouped by rule.
 
+:::info
+In this view, the rules that reported the highest number of findings are at the top of the page. This sorting of findings helps you to easily see which rules have the highest efficiency or generate a lot of noise in your code.
```
Comment (adamkvitek) [inline docs/semgrep-app/findings.md:145]: @raghavjain3 let me know if this is valid please.

File: docs/release-notes/february-2023.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+- When you now click on a hyperlink header with the rule name in [Semgrep Registry](https://semgrep.dev/explore), the link opens a new tab with the rule in either the Semgrep Playground (if you are logged out) or in Semgrep Editor (if you are logged in). This can help you to keep the Registry page open but look at or modify the rule.
+
+## Documentation
+
+- Slight improvements to relevancy in Semgrep Docs’s search bar.
+- Updates to **Supported Languages** > **Semgrep Supply Chain** and **Semgrep Code** > **Semgrep Pro Engine**.
+- Updates to **Semgrep Code** > **Alerts and Notifications** to consolidate all methods to send and receive scan data, such as findings.
+- Updates to **Pricing and Billing** to reflect the differences between Semgrep OSS Engine and Semgrep Code.
+- New documentation for Semgrep Code.
```
Comment (adamkvitek) [inline docs/release-notes/february-2023.md:79]: TODO

Comment (s-santillan) [reply docs/release-notes/february-2023.md:79]: Semgrep Code is an entirely new segment, you can see it on the left sidebar, there's a new doc for "semgrep code"

Comment (adamkvitek) [reply docs/release-notes/february-2023.md:79]: @s-santillan got it. Would you agree with this rephrasing:

```suggestion
- Added new documentation category Semgrep Code.
```

File: docs/semgrep-app/findings.md (hunk 2 above)
Comment (raghavjain3) [reply docs/semgrep-app/findings.md:145]: looks good!

Comment (adamkvitek) [reply docs/semgrep-app/findings.md:145]:
```suggestion
In the **group by rule** view, the rules that reported the highest number of findings are at the top of the page. This sorting of findings helps you to easily see which rules have the highest efficiency or generate a lot of noise in your code.
```

File: docs/release-notes/february-2023.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+# February 2023
+
+## Important update
+
+- Semgrep CLI is now officially renamed to **Semgrep OSS Engine**. Note that Semgrep CLI is an interface, not a product. As an interface, you can use the CLI as an interface for several products. For example the OSS, Semgrep Code, Semgrep Supply Chain.
+- Team tier rules are now renamed to Pro Rules. The **Pro rules** are created by r2c and targeted for security and software engineers who need accurate findings. These rules were previously called Team tier rules. As of this update, these rules are officially called the **[Pro rules](/semgrep-code/pro-rules/)** and are available with the [Team or higher tier](https://semgrep.dev/pricing).
+- DeepSemgrep has been bundled with other functionalities to offer you **Semgrep Pro Engine**. Semgrep Pro Engine is fully available for [Team or higher tier](https://semgrep.dev/pricing) users. See the [DeepSemgrep → Semgrep Pro Engine](#deepsemgrep--semgrep-pro-engine) update below for more details.
+
+## Semgrep CLI and CI → Semgrep OSS Engine
```
Comment (adamkvitek) [inline docs/release-notes/february-2023.md:18]:
```suggestion
## Semgrep CLI → Semgrep OSS Engine
```

File: docs/release-notes/february-2023.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+description: >-
+  Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+---
+
+# February 2023
+
+## Important update
+
+- Semgrep CLI is now officially renamed to **Semgrep OSS Engine**. Note that Semgrep CLI is an interface, not a product. As an interface, you can use the CLI as an interface for several products. For example the OSS, Semgrep Code, Semgrep Supply Chain.
```
Comment (s-santillan) [inline docs/release-notes/february-2023.md:14]:
```suggestion
- Semgrep CLI is now officially renamed to **Semgrep OSS Engine**. You can use Semgrep CLI for several products, such as Semgrep OSS, Semgrep Code, and Semgrep Supply Chain.
```

My "Semgrep CLI is an interface, not a product" note in the draft was an internal note, because it spells out to "Semgrep Command line interface is an interface" i.e. redundant.

File: docs/release-notes/february-2023.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+- Team tier rules are now renamed to Pro Rules. The **Pro rules** are created by r2c and targeted for security and software engineers who need accurate findings. These rules were previously called Team tier rules. As of this update, these rules are officially called the **[Pro rules](/semgrep-code/pro-rules/)** and are available with the [Team or higher tier](https://semgrep.dev/pricing).
```
Comment (s-santillan) [inline docs/release-notes/february-2023.md:15]:
```suggestion
- Team tier rules are now renamed to Pro Rules. **Pro rules** are created by r2c for security and software engineers who need accurate findings. These rules were previously called Team tier rules. As of this update, these rules are officially called the **[Pro rules](/semgrep-code/pro-rules/)** and are available with the [Team or higher tier](https://semgrep.dev/pricing).
```

**Review by s-santillan — APPROVED** [review]
Left some comments!

File: docs/release-notes/february-2023.md (hunk 5 above)
Comment (adamkvitek) [reply docs/release-notes/february-2023.md:14]: I knew it is internal, but have been thinking that it was really cool to include it. Thanks for the suggestion! @s-santillan
